### ISSUE TITLE
Issue with "Aligning Items in a Flex Container": Description of column-gap and row-gap seems incorrect

### ISSUE THREAD

**Issue by bryanyanay** [mdn/content#14415]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container

#### What information was incorrect, unhelpful, or incomplete?
The article right now states: "The column-gap property creates gaps between items on the main axis. The row-gap property creates gaps between flex lines, when you have flex-wrap set to wrap."

I believe this is only correct for when the main axis is horizontal. When the main axis is vertical, it seems that column-gap would create gaps between flex lines, and row-gap would then create gaps between items on the main axis.

My test in the interactive example seems to confirm this belief (see "Did you test this? If so, how?").

#### Specific section or headline?
This is in the "Creating gaps between items" section.

#### What did you expect to see?
The description of column-gap and row-gap should be corrected.

#### Did you test this? If so, how?
I tested it in the interactive example:
![image](https://user-images.githubusercontent.com/74737261/160492884-336bbef8-52ce-438e-8892-00c1cdd621af.png)
You can see that here, the main axis is vertical. The row-gap is still separating rows, but now that would be separating flex items on the main axis, and the column-gap is still separating columns, but now that would be separating flex lines, contrary to what the article currently states. 

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/css_flexible_box_layout/aligning_items_in_a_flex_container`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/css_flexible_box_layout/aligning_items_in_a_flex_container/index.md
* Last commit: https://github.com/mdn/content/commit/7f69f6ff587b5d9d27cfb3fc87ab237aa7cfc63e
* Document last modified: 2022-03-25T12:45:30.000Z

</details>

**Comment by Anurella** [comment]
@sideshowbarker You can assign this to me. Will it be okay to add that the direction of the main axis influences the column and row gap syntax?

**Comment by sideshowbarker** [comment]
> Will it be okay to add that the direction of the main axis influences the column and row gap syntax?

I think so, yes

**Comment by Anurella** [comment]
Hello @bryanyanay,  a better breakdown is that the column gap has effect on items from left to right (horizontally) while the row-gap is a top to bottom effect ( vertically ), the main axis does not affect the syntax.  

Your example is great to test this out, try setting either the row or column to 0 and notice the effect, also change the direction of the axis as you test. 

Would love to hear your findings

**Comment by Anurella** [comment]
> > Will it be okay to add that the direction of the main axis influences the column and row gap syntax?
> 
> I think so, yes

I carried out the examples and noticed that the main axis does not influence the syntax as earlier thought. I have added the breakdown explanation as commented above in  a `note` tag. You can make suggestions when I make a pull request.

Thank you

**Comment by OnkarRuikar** [comment]
@bryanyanay `flex-direction: column` does not make the "main axis" vertical. The `writing-mode` does. Your code is not making the "main axis" vertical. The items are still being laid along the x-axis, the columns are laid left to right.

Consider this code:
![use writing-mode](https://i.imgur.com/SANFAhl.png)
When the main axis is vertical, columns are horizontal and `column-gap` is horizontal, i.e. between an element and the element above it. Similarly, row-gap follows the cross axis. 

The items are laid using both `flex-flow`(direction and wrap) and `writing-mode`. [[ref](https://www.w3.org/TR/css-flexbox-1/#box-model)]

**Comment by Anurella** [comment]
@OnkarRuikar thank you for the insight

**Comment by bryanyanay** [comment]
@OnkarRuikar You're right, I didn't consider different writing modes.

**Comment by bryanyanay** [comment]
> Hello @bryanyanay, a better breakdown is that the column gap has effect on items from left to right (horizontally) while the row-gap is a top to bottom effect ( vertically ), the main axis does not affect the syntax.
> 
> Your example is great to test this out, try setting either the row or column to 0 and notice the effect, also change the direction of the axis as you test.
> 
> Would love to hear your findings

@Anurella Yep, I think your description is clearer and more on point. Perhaps we should include a note saying that in other writing modes (e.g., vertical writing mode) this changes though.
